### ISSUE TITLE
iframe.contentWindow evasion can leak Object.apply in stack traces

### ISSUE THREAD

**Issue by prescience-data** [berstend/puppeteer-extra#318]
Similar to #209 there is instances in the wild searching for: `Object.apply (<anonymous>` in thrown error stack traces.

#### Real browser:

```js
Error: Failed to execute 'createElement' on 'Document': The tag name provided ([redacted]) is not a valid name.
    at test.js?660921:2
```

#### Puppeteer with Extra Stealth 2.6.1

```js
Error: Failed to execute 'createElement' on 'Document': The tag name provided ([redacted]) is not a valid name.
    at Object.apply (<anonymous>:80:31)
    at test.js?103622:2
```

Second case fails the vendor detection test.

**Comment by momala454** [comment]
careful to not remove all Object.apply, which could be detected by purposely creating a stack containing Object.apply

**Comment by Niek** [comment]
Sample test code:
```js
try {
    document["createElement"](0);
} catch (e) {
    console.log(e.stack)
}
```

**Comment by prescience-data** [comment]
> Sample test code:
> 
> ```js
> try {
>     document["createElement"](0);
> } catch (e) {
>     console.log(e.stack)
> }
> ```

Sample code including the observed detection:

``js
var detection;
var stack;
try {
    document.createElement(0);
} catch (err) {
    try {
        stack = err.stack.split('\n');
        detection = (stack.length >= 2) ? !!stack[1].match(/Ob[cej]{3}t\.a[lp]{3}y[\(< ]{3}an[oynm]{5}us>/) : 'Normal stack trace.';
    } catch (errOfErr) {
        detection = 'Stack trace hijacked.';
    }
}
```

**Comment by berstend** [comment]
This is leveraging the ancient/shitty `iframe.contentWindow` evasion, which hasn't been updated yet to use the new stealth utilities: https://github.com/berstend/puppeteer-extra/blob/12fe3afd927327f7624e6ac0b255db02c586ac4d/packages/puppeteer-extra-plugin-stealth/evasions/iframe.contentWindow/index.js#L95

**Comment by berstend** [comment]
Workaround for the time being: Disabling the `iframe.contentWindow` evasion

**Comment by berstend** [comment]
Fixed in `puppeteer-extra-plugin-stealth@2.6.5`
